.bumpversion.cfg

```diff
@@ -3,6 +3,8 @@ current_version = 0.6.2
 commit = True
 tag = True
 
+[bumpversion:file:setup.cfg]
```

Comment:
is there a reason this got added? It should be picked up automatically from the auto-generated version.

Reply:
c.f. below